core/domain/services/assets/WordPressPluginsPageAssetManager.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace EventEspresso\core\domain\services\assets;
+
+use DomainException;
+use EventEspresso\core\services\assets\AssetManager;
+
+/**
+ * WordPressPluginsPageAssetManager
+ * assets for WP Plugins admin page
+ *
+ * @package EventEspresso\core\domain\services\assets
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class WordPressPluginsPageAssetManager extends ReactAssetManager
+{
+    const DOMAIN = 'wpPluginsPage';
+
+    const ASSET_HANDLE_WP_PLUGINS_PAGE = AssetManager::EE_NAMESPACE . WordPressPluginsPageAssetManager::DOMAIN;
+
+    /**
+     * @throws DomainException
+     */
+    public function enqueueAssets()
+    {
+        if ($this->verifyAssetIsRegistered(WordPressPluginsPageAssetManager::ASSET_HANDLE_WP_PLUGINS_PAGE)) {
```

Comment:
:-1:

Reply:
I like when code is expressive, like

```
if (assetHandlerIsRegistered) then
```

I wrote that "I like" because it's a personal preference, not a best practice

Reply:
bah... auto-formatting gone wrong... will fix thnx

Reply:
Yes, in PHP minimizing the number of variables is always preferred.

Reply:
I :heart: @tn3rb's in-depth comments, so much to learn from;
my comment wasn't a suggestion per se, just an observation/ thought outloud so I am ok with that being ignored